src/pytorch/neighbors/__init__.py

```diff
@@ -1,5 +1,9 @@
 '''
 Neighbor operations
 '''
+import os.path
+import torch
 
-from NNPOps.neighbors.getNeighborPairs import getNeighborPairs
\ No newline at end of file
+torch.ops.load_library(os.path.join(os.path.dirname(os.path.dirname(__file__)), 'libNNPOpsPyTorch.so'))
```

Comment:
This was intended, as libNNPOps is at the parent directory with respect to neighbors.
In any case, this line sometimes tries to look in the src folder instead of the installation one. I thus went back to your idea of looking for the library in all the folders returned by getsitepackages().

Now tests pass on my machine and the original issue is fixed. Lets wait for the CI...